lib/perf_hooks.js

```diff
@@ -471,6 +471,10 @@ class Performance extends PerformanceObserverEntryList {
     this[kClearEntry]('function', name);
   }
 
+  clearEntries(name) {
+    this[kClearEntry](name);
```

Comment:
do we still need the Symbol now that this is exposed? Should we perform any validation?

Reply:
The symbol form takes the additional `name` argument, which is not strictly necessary here, so I'd rather keep that. Validation is unnecessary since it would just be a non-op if some other value is passed.